infrastructure/docker/services/frontend/etc/nginx/nginx.conf

```diff
@@ -39,6 +39,10 @@ http {
         # secure this page (by user IP address, with a password, for example)
         location ~ ^/php-fpm-status$ {
             deny all;
+            fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+            fastcgi_index index.php;
```

Comment:
Do we still need `SCRIPT_NAME` override in the location block with these config ?

Reply:
yes, because we override the SCRIPT_NAME for "regular" HTTP request (like on /post, or /contact)